client.lua

```diff
@@ -1,3 +1,4 @@
-RegisterNetEvent("ND:setCharacter", function(character)
+AddEventHandler("ND:characterLoaded", function(character)
     TriggerServerEvent("ND_npwd:refresh", character.id)
-end)
\ No newline at end of file
+    print(character.id)
+end)

```

Comment:
There's a server event for characterLoaded https://ndcore.dev/core/server/events#ndcharacterloaded